modules/minimum-required-permissions-upi-azure.adoc

```diff
@@ -0,0 +1,236 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="minimum-required-azure-permissions_{context}"]
```

Comment:
This ID is the same as the one in modules/minimum-required-permissions-ipi-azure.adoc. Please change both module IDs so that they match the corresponding filename. This one should be: `[id="minimum-required-permissions-upi-azure_{context}"]`

Thanks!